ntex-server/src/net/test.rs

```diff
@@ -131,6 +140,16 @@ impl TestServer {
         let tcp = net::TcpListener::from(socket);
         tcp.local_addr().unwrap()
     }
+
+    /// Get access to the running Server
+    pub fn server(&self) -> &Server {
+        &self.server
+    }
+
+    /// Get mutable access to the running Server
```

Comment:
I don't think this method is needed. Server could be cloned. or let's just change Server::process() to use &self

Reply:
Forgot that it's just a handle.